### PR TITLE
[New recipe] Kottan 0.10.8

### PR DESCRIPTION
**Pull request by andimachovec**
new recipe for the Kottan BMessage file viewer and editor

### CODE REVIEW

File: haiku-apps/kottan/kottan-0.10.8.recipe [hunk 1]
```diff
@@ -0,0 +1,43 @@
+SUMMARY="A viewer and editor for flattened BMessages"
+DESCRIPTION="Kottan lets you view and edit the contents of files that contain flattened BMessages"
```
Comment (Begasus) [inline haiku-apps/kottan/kottan-0.10.8.recipe:2]: Keep DESCRIPTION line up to 80 characters:
```diff
- DESCRIPTION="Kottan lets you view and edit the contents of files that contain flattened BMessages"
+ DESCRIPTION="Kottan lets you view and edit the contents of files that contain \
+flattened BMessages"
```

File: haiku-apps/kottan/kottan-0.10.8.recipe [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+	
+BUILD_PREREQUIRES="
```
Comment (Begasus) [inline haiku-apps/kottan/kottan-0.10.8.recipe:26]: Trailing whitespace on the newline, this should be deleted (no newline between BUILD_REQUIRES and BUILD_PREREQUIRES):
```diff
	"
-
BUILD_PREREQUIRES="
```

**Comment by Begasus** [comment]
Builds fine on 32bit :+1:

**Comment by andimachovec** [comment]
Thanks for the review Begasus. It took me a while to get the thing with the trailing whitespace right, but I finally seem to have done it correctly ;-)

**Comment by Begasus** [comment]
Let's see if the buildbot agrees ;) LGTM Thanks! :+1:

**Pull request merged**